src/fping.c

```diff
@@ -2384,6 +2402,10 @@ int wait_for_reply(int64_t wait_time)
     unsigned short seq;
     int ip_header_tos = -1;
     int ip_header_ttl = -1;
+    // ICMP Timestamp
+    long ip_header_otime_ms = -1;
```

Comment:
I would prefer an exact length type, e.g., `int64_t` or `int32_t`. Or perhaps even `uint32_t` with an initialization to `0xffffffff`.

The timestamps are defined to be contained in a 32 bit field each. One day has 86,400,000ms (or 86,401,000 with a leap second). Values above cannot be valid, and only conform to RFC 792 when the highest bit is set. Setting the highest bit would result in a negative value when interpreting the 32 bit field as a signed integer. As such, a timestamp may contain a value of `-1`.

Reply:
Has been changed to uint32_t as this is sufficient and 0xffffffffff is not required.